Bottom: 00e1df4bcbc33b38df0e85b87fbbabd2da64d772
Top:    1156af15371b9307f7a9cf12d291ba35e3e631c6
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-09 17:19:03 -0700

Refresh of build-preload

---

diff --git a/Makefile.am b/Makefile.am
index 901e8ee..9bf6070 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,7 @@
 INCLUDES = -I$(srcdir)/include
 
-lib_LTLIBRARIES = src/librdmacm.la src/librspreload.la
+lib_LTLIBRARIES = src/librdmacm.la
+noinst_LTLIBRARIES  = src/librspreload.la
 
 ACLOCAL_AMFLAGS = -I config
 AM_CFLAGS = -g -Wall -D_GNU_SOURCE -DSYSCONFDIR=\"$(sysconfdir)\" -DRDMADIR=\"@rdmadir@\"
